wasm/store_test.go

```diff
@@ -0,0 +1,22 @@
+package wasm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetModuleInstance(t *testing.T) {
+	name := "test"
+
+	// 'jit' and 'wazeroir' package cannot be used because of circular import.
+	// Here we will use 'nil' instead. Should we have an Engine for testing?
```

Comment:
thanks for this comment, and yeah actually this is problematic. We could solve this as part of #83. Meanwhile, I'm fine using nil for this test.